features/foutafhandeling.feature

```diff
@@ -155,9 +154,9 @@ Functionaliteit: Afhandeling van fouten
 
   Scenario: onjuiste waarde in request header Content-Crs
     Als /panden wordt gezocht met de niet-ondersteunde waarde voor header Content-Crs "epsg:4326"
-    Dan is de http status code van het antwoord 415
+    Dan is de http status code van het antwoord 406
```

Comment:
Dit komt niet overeen met het voorstel om een 400 terug te geven in deze situatie. 
Zie BAG issue https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/370.